test/plugins/webhooks/github.test.js

```diff
@@ -506,21 +583,6 @@ describe('github plugin test', () => {
                     });
                 });
             });
-
-            describe('other change pull request', () => {
```

Comment:
We removed it because `parseHook` does not handle any other actions for a pull request except opened, reopened, synchronized, and closed.